libcontainer/container_linux.go

```diff
@@ -1889,17 +1889,12 @@ func (c *linuxContainer) currentState() (*State, error) {
 		ExternalDescriptors: externalDescriptors,
 	}
 	if pid > 0 {
-		for _, ns := range c.config.Namespaces {
-			state.NamespacePaths[ns.Type] = ns.GetPath(pid)
-		}
 		for _, nsType := range configs.NamespaceTypes() {
 			if !configs.IsNamespaceSupported(nsType) {
 				continue
 			}
-			if _, ok := state.NamespacePaths[nsType]; !ok {
-				ns := configs.Namespace{Type: nsType}
-				state.NamespacePaths[ns.Type] = ns.GetPath(pid)
-			}
+			ns := configs.Namespace{Type: nsType}
+			state.NamespacePaths[ns.Type] = ns.GetPath(pid)
```

Comment:
I am not sure why this code was written the way it was -- I checked it comes from commits 556f798a19ecf and 78f5148c67498b, and I'm sure even @crosbymichael does not remember anymore.